File: unit-3/sprint-1/day-4_Fragment Transaction/assignments/Dynamic Fragment/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;
import android.os.Bundle;
import android.view.View;
import android.widget.FrameLayout;
import android.widget.LinearLayout;
import android.widget.TextView;
public class MainActivity extends AppCompatActivity {
    FrameLayout flContainer;
    FragmentManager fragmentManager;
    LinearLayout linearLayout;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        linearLayout = findViewById(R.id.tvLinearLayout);
        flContainer = findViewById(R.id.flContainer);
        fragmentManager = getSupportFragmentManager();
        linearLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                replace();
            }
        });
    }

    private void replace() {
        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
        FragmentA fragmentName = new FragmentA();
        fragmentTransaction.replace(R.id.flContainer,fragmentName,"fragmentA").commit();
    }
}

File: Trive- Revision Placement/Android Short Assignment- 1/app/src/main/java/com/example/communication/MainActivity.kt
package com.example.communication

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity(){
    private var fragmentA: FragmentA? = null
    private var fragmentB: FragmentB? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        fragmentA = FragmentA()
        fragmentB = FragmentB()
        supportFragmentManager.beginTransaction()
            .replace(R.id.container_a, fragmentA!!)
            .replace(R.id.container_b, fragmentB!!)
            .commit()
    }
}